tests/e2e/fill-incorrect-address-input-values.test.ts

```diff
@@ -0,0 +1,92 @@
+import {
+  closeBrowser,
+  openBrowser,
+  text,
+  click,
+  screenshot,
+  $,
+  evaluate,
+  goto,
+  currentURL,
+  below,
+} from "taiko";
+
+import { authenticateToConnect, printStep } from "./utils";
+import * as locales from "@content/locales";
+import { CONFIG_VARIABLES } from "@src/configs/config-variables";
+
+describe("Profile happy path", () => {
+  jest.setTimeout(60000);
+
+  const isStagingEnv = (
+    process.env.CONNECT_PROFILE_URL || CONFIG_VARIABLES.connectProfileUrl
+  ).includes("staging");
+
+  const localizedStrings = {
+    newAddress: locales.en["/account/profile/addresses/new"],
+    errors: locales.en["errors"],
+  };
+
+  beforeAll(async () => {
+    if (isStagingEnv) {
+      await openBrowser({
+        args: [
+          "--window-size=1440,1000",
+          "--no-sandbox",
+          "--start-maximized",
+          "--disable-dev-shm",
+        ],
+        headless: false,
+        observe: true,
+        observeTime: 1000,
```

Comment:
Is it intended to do it in observe mode ?

Reply:
I did as we do on every other e2e test.
I tried without it but it failed ont the `for await` part and it failed 1 time on 5 with an `observeTime` of 500ms, so I let 1000ms.

Reply:
Alright, sometimes we don't have any other choice 👍